src/Common/src/Interop/Unix/libcurl/Interop.libcurl_types.cs

```diff
@@ -166,8 +175,41 @@ internal struct curl_version_info_data
             private int versionNum;
             private unsafe char *host;
             internal int features;
+            internal IntPtr ssl_version;
+        }
+
+        internal static partial class CurlSslBackend
```

Comment:
This is an awful lot of code in a "types" file.  It should probably live elsewhere.